vdb-bench-assembly/plugins/vdb-bench-dataservice/dsPageService.js

```diff
@@ -167,8 +167,8 @@
         };
         pages[service.SERVICESOURCE_NEW_PAGE] = {
             id: service.SERVICESOURCE_NEW_PAGE,
-            title: 'New Service-source',
-            icon: 'fa-plus',
+            title: 'Configure Source',
```

Comment:
'Configure source' could imply configuring an existing source (as in edit source) as well as refer to configuring a new source so I think that the word 'new' needs to appear in there, eg. Configure New Source.